pittapi/library.py

```diff
@@ -144,8 +144,7 @@ def hillman_total_reserved():
     count["Total Hillman Reservations"] = total_records
     return count
 
-
-def reserved_hillman_times():
+def reserved_hillman_times() -> List[Dict[str, Union[str, List[str]]]]:
```

Comment:
In Python, the style convention is that there should be two empty lines between functions (unless you're in a class, in which case there should be one empty line between methods)